packages/orbit-components/src/Popover/__tests__/index.test.js

```diff
@@ -66,14 +66,16 @@ describe("Popover", () => {
   });
 
   it("should have offset", () => {
-    render(
+    const { debug } = render(
       <Popover opened offset={{ top: 10, left: 20 }} content="kek">
         <Button type="secondary" size="small">
           Cancel
         </Button>
       </Popover>,
     );
 
+    debug();
```

Comment:
I know about `draft`, i thought it's ready, but then it's started to fail again and i know why, so i changed it to `WIP`, in order to prevent you from reviewing not finished PR :D

Reply:
thanks a lot for suggestion :) applied `inline snapshot` to `Mobile` and removed that `media` check for `Popover`, seems like a redundant check, I think it's enough to check if the `offset` applied